custom_components/tuya_local/devices/aspen_fan.yaml

```diff
@@ -0,0 +1,95 @@
+name: Aspen Fan
+primary_entity:
+  entity: fan
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+    - id: 2
+      type: string
+      name: preset_mode
+      mapping:
+        - dps_val: in
+          value: cool
+        - dps_val: out
+          value: exhaust
+        - dps_val: exch
+          value: exchange
+    - id: 3
+      type: integer
+      name: speed
+      mapping:
+        - dps_val: 1
+          value: low
+        - dps_val: 2
+          value: medium
+        - dps_val: 3
+          value: high
+secondary_entities:
+  - entity: climate
```

Comment:
I think its better to have either Fan + current temperature (sensor) + temperature (number), or a climate entity.

Reply:
I mentioned this in the response below but the reason for this is the functionality of id:101. If this is on it functions like a "cooling device" it allows you to set a "desired" temperature and the fan will run until reaches this temperature, to me this seemed more like the functionality of a climate device. At the same time this is not a feature I used much so I wanted an alternative for control. Is it an issue to have both? If so which one do you think I should remove?